### PR TITLE
Cookbook example: building complex static pages with widgets

### PR DESCRIPTION
**Pull request by ljubar**
| Q                         | A
| ------------------------- | ---
| Bug fix?                  |no
| Changelog update required |no
| License                   | AGPLv3

Docs entry written

### CODE REVIEW

File: docs/cookbook/templates/complex_static_pages_widgets.rst [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+.. code-block:: twig
+
+    {% container 'frontpage_sidebar' with {
+    'width': 300,
+    'height': 600,
+    'styles': 'border: solid 1px red',
+    'cssClass': 'css_class_name',
+    'data': {'custom-key': value}
+    }%}
```
Comment (ahilles107) [inline docs/cookbook/templates/complex_static_pages_widgets.rst]: space between `}` and `%}` is missing.

File: docs/cookbook/templates/complex_static_pages_widgets.rst [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+.. code-block:: twig
+
+    {% container 'frontpage_sidebar'%}
+    ...
+    {% endcontainer %}
+
+This will generate block-level element on web page (div) with special class - it is used in live editing to define dynamic canvas where editors can work with widgets.
+
+Containers can have none, one or more widgets - for example, think of frontpage sidebar; if it is defined as container in template, then editors can create several widgets in that container. From time to time they can turn on widget that highlights for example some recent interview; or can put some important announcement on top of the sidebar; etc - and all that without any changes in templates, only by playing wiith live editing mode.
```
Comment (ahilles107) [inline docs/cookbook/templates/complex_static_pages_widgets.rst]: You can add something about default HTMLWidget and describe how it works and how we can use it for live content change.

File: docs/cookbook/templates/complex_static_pages_widgets.rst (hunk 1 above)
Comment (ljubar) [inline docs/cookbook/templates/complex_static_pages_widgets.rst]: added

File: docs/cookbook/templates/complex_static_pages_widgets.rst (hunk 2 above)
Comment (ljubar) [inline docs/cookbook/templates/complex_static_pages_widgets.rst]: added paragraph at the end

**Pull request merged**